xclim/data/fr.json

```diff
@@ -942,5 +942,29 @@
+  },
+  "COLD_AND_DRY_DAYS": {
+    "title": "Durée totale en jours des périodes froides et seches",
+    "abstract": "Durée totale en jours où la température est en dessous du 25eme percentile et les précipitations en dessous du 25eme percentile.",
+    "description": "Durée totale {freq:f} en jours où la température est en dessous du 25eme percentile et les précipitations en dessous du 25eme percentile. Les percentiles sont calculés à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence. Concernant les precipitations les percentiles ne sont calculés que pour des valeurs au dessus de 1 mm.",
+    "long_name": "Durée totale {freq:f} en jours des périodes froides et seches"
+  },
+  "WARM_AND_DRY_DAYS": {
+    "title": "Durée totale en jours des périodes chaudes et seches",
+    "abstract": "Durée totale en jours où la température est au dessus du 75eme percentile et les précipitations en dessous du 25eme percentile.",
+    "description": "Durée totale {freq:f} en jours où la température est au dessus du 75eme percentile et les précipitations en dessous du 25eme percentile. Les percentiles sont calculés à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence. Concernant les precipitations les percentiles ne sont calculés que pour des valeurs au dessus de 1 mm.",
+    "long_name": "Durée totale {freq:f} en jours des périodes chaudes et seches"
+  },
+  "WARM_AND_WET_DAYS": {
+    "title": "Durée totale en jours des périodes chaudes et pluvieuses",
+    "abstract": "Durée totale en jours où la température est au dessus du 75eme percentile et les précipitations au dessus du 75eme percentile.",
+    "description": "Durée totale {freq:f} en jours où la température est au dessus du 75eme percentile et les précipitations au dessus du 75eme percentile. Les percentiles sont calculés à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence. Concernant les precipitations les percentiles ne sont calculés que pour des valeurs au dessus de 1 mm.",
+    "long_name": "Durée totale {freq:f} en jours des périodes chaudes et pluvieuses"
+  },
+  "COLD_AND_WET_DAYS": {
+    "title": "Durée totale en jours des périodes froides et pluvieuses",
+    "abstract": "Durée totale en jours où la température est en dessous du 25eme percentile et les précipitations au dessus du 75eme percentile.",
+    "description": "Durée totale {freq:f} en jours où la température est en dessous du 25eme percentile et les précipitations au dessus du 75eme percentile. Les percentiles sont calculés à partir d'une fenêtre de 5 jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence. Concernant les precipitations les percentiles ne sont calculés que pour des valeurs au dessus de 1 mm.",
+    "long_name": "Durée totale {freq:f} en jours des périodes froides et pluvieuses"
```

Comment:
Also, in the docstring it is written that the quartiles are _per month_, while here and in the ICCLIM yaml they are _daily_. 

In any case, I removed the percentile explanation from the `description` field because the indicator does not know that. The percentiles are user-provided so they could be computed in many ways, we shouldn't assume how. In theory, that information will be available through the `xclim_history` attribute. And there is no reason the french metadata assumes more than the english anyway ;) !